youtube_dl/extractor/twitter.py

```diff
@@ -44,6 +57,20 @@ def _real_extract(self, url):
                 unescapeHTML(self._search_regex(
                     r'data-player-config="([^"]+)"', webpage, 'data player config')),
                 video_id)
+            if 'playlist' not in config:
+                if 'vmapUrl' in config:
+                    webpage = self._download_webpage(config['vmapUrl'], video_id + ' (xml)')
```

Comment:
Using ```self._download_xml``` is better here. And by the way, it's uncommon to use prefixes or suffixes in the second field.